manticore/core/cpu/abstractcpu.py

```diff
@@ -769,16 +770,15 @@ def decode_instruction(self, pc):
                 if isinstance(c, Constant):
                     c = c.value
                 else:
-                    logger.error('Concretize executable memory %r %r', c, text)
+                    logger.error('Concretize executable memory {!r} {!r}'.format(c, text))
```

Comment:
This will make the formatting even if we are not in the debuglevel that prints errors